Library/Formula/vala.rb

```diff
@@ -10,10 +10,11 @@ class Vala < Formula
     sha256 "b55053adbebaf5d9d016002fb8d2a26b72398649e5a54c5621a68a9e62a31e62" => :mountain_lion
   end
 
-  devel do
-    url "https://download.gnome.org/sources/vala/0.29/vala-0.29.3.tar.xz"
-    sha256 "687f89c404cec528bc6c8e80d80146be0a42aa30f60475fcaefad37bd12949fe"
-  end
+  # currently there is no development release available
```

Comment:
I'm afraid we very deliberately kick stuff out of formulae if it isn't being used :/. Commented out blocks tend to encourage people to do the same thing with other elements on updates, like our bottle blocks, sadly.